methods.py

```diff
@@ -134,6 +134,20 @@ def parse_cg_file(fname, uniforms, sizes, conditionals):
     fs.close()
 
 
+def get_cmdline_bool(option, default):
+    """We use `ARGUMENTS.get()` to check if options were manually overridden on the command line,
+    and SCons' _text2bool helper to convert them to booleans, otherwise they're handled as strings.
+    """
+    from SCons.Script import ARGUMENTS
```

Comment:
Inline import are not cheap (they have a relatively high cost even if the module has already been imported)
```shell
$ python -m timeit -s "from SCons.Script import ARGUMENTS" "pass"
100000000 loops, best of 3: 0.00505 usec per loop
$ python -m timeit -s "pass" "from SCons.Script import ARGUMENTS"
1000000 loops, best of 3: 0.433 usec per loop
```

You should instead consider doing the imports at top level ;-)

Reply:
Thanks, fixed.